AUTH.md

```diff
@@ -41,7 +41,9 @@ where `client`, `tenant`, `subscriptionId`, and `key` or `pfxCertificatePath` an
 
 ## Using Proxy in authentication
 
-This method will help you specify the proxy credentials by two steps when you want to create a authenticated client. First, it requires an instance of the `AzureCredentials` class from either of the above approaches:
+This method will help you specify the proxy credentials by two steps when you want to create an authenticated client. 
```

Comment:
This approach will setup proxy credentials with two steps to create an authenticated client.